core/src/main/java/com/google/instrumentation/stats/Stats.java

```diff
@@ -26,7 +26,7 @@
    * Returns the default {@link StatsContextFactory}.
    */
   public static StatsContextFactory getStatsContextFactory() {
-    return statsManager.getStatsContextFactory();
+    return statsManager == null ? null : statsManager.getStatsContextFactory();
```

Comment:
There is already an issue requesting a no-op implementation: #41

/cc @zhangkun83